### PR TITLE
Enabling PublishBuildArtifact task in RM file share scenerio

### PR DESCRIPTION
**Pull request by SunnyKumarR**
Enabling PublishBuildArtifact task in RM file share scenerio

### CODE REVIEW

File: src/Misc/layoutbin/en-US/strings.json [hunk 1]
```diff
@@ -105,6 +105,7 @@
   "FailedToReadFile": "Failed to read {0}. Error : {1}.",
   "FailedToReplaceAgent": "Failed to replace the agent.  Try again or ctrl-c to quit",
   "FailToRemoveGitConfig": "Unable to remove \"{0}\" from the git config. To remove the credential, execute \"git config --unset-all {0}\" from the repository root \"{1}\".",
+  "AssociateArtifactCommandNotSupported": "Associating server artifact is not supported in {0}.",
```
Comment (GitHubSriramB) [inline src/Misc/layoutbin/en-US/strings.json]: keep the keys sorted

File: src/Misc/layoutbin/en-US/strings.json [hunk 2]
```diff
@@ -459,6 +460,7 @@
   "UpdateInProgress": "Agent update in progress, do not shutdown agent.",
   "UpgradeToLatestGit": "In order to get better Git experience, upgrade your Git to at least version '{0}', your current git version is '{1}'.",
   "UploadArtifact": "Upload Artifact",
+  "UploadArtifactCommandNotSupported": "Uploading artifact is not supported in {0}.",
```
Comment (TingluoHuang) [inline src/Misc/layoutbin/en-US/strings.json]: should this be `Uploading server artifact is not supported in {0}.`?

Comment (SunnyKumarR) [inline src/Misc/layoutbin/en-US/strings.json]: Sure. Modifying it.

File: src/Misc/layoutbin/en-US/strings.json (hunk 1 above)
Comment (SunnyKumarR) [inline src/Misc/layoutbin/en-US/strings.json]: Done

**Pull request merged**